### PR TITLE
Updated Checkout Button Enablement in checkbox component

### PR DESCRIPTION
**Pull request by Shopgate-Travis**
## Description

Checkout Button enabled/disabled state was not updating correctly when adding configurable terms and conditions checkboxes. In PR I updated componentDidUpdate to include function to update orderability.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

## To-Test:

Deployed rc to shop no: 31770 with a configuration for product id's: 1592032329770, 1596104278058, and 1596109815850

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopgate/terms-and-conditions",
-  "version": "1.1.0",
+  "version": "1.2.0",
```
Comment (Carsten89) [inline frontend/package.json]: in other places you changed to version to 1.1.1

File: frontend/package.json [hunk 2]
```diff
@@ -30,14 +30,16 @@
...
-    "react-transition-group": "^2.2.1"
+    "react-transition-group": "^2.2.1",
+    "react-hot-loader": "^3.1.3",
+    "react-transform-catch-errors": "^1.0.2"
   },
   "peerDependencies": {
-    "@shopgate/pwa-common": "^6.2.1",
-    "@shopgate/pwa-common-commerce": "^6.2.1",
-    "@shopgate/pwa-ui-shared": "^6.2.1"
+    "@shopgate/pwa-common": "^6.5.4",
```
Comment (Carsten89) [inline frontend/package.json]: is it really needed to to increase the min required PWA version here?
We should keep the min version as low as possible

File: frontend/package.json (hunk 1 above)
Comment (Shopgate-Travis) [reply frontend/package.json]: Sorry I think I didn't end up pushing the updated version for this package file.

Comment (DannyShopgate) [reply frontend/package.json]: Sorry, version fixed.

File: frontend/package.json (hunk 2 above)
Comment (DannyShopgate) [reply frontend/package.json]: Reverted to ^6.2.1

**Pull request merged**